src/components/ButtonPannel.js

```diff
@@ -1,42 +1,48 @@
+/* eslint-disable react/require-default-props */
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPannel = () => (
+const ButtonPannel = ({ clickHandler }) => (
   <div className="ButtonPannel">
     <div className="group1">
-      <Button name="AC" color="e0e0e0" />
-      <Button name="+/-" color="e0e0e0" />
-      <Button name="%" color="e0e0e0" />
-      <Button name="÷" />
+      <Button onClick={clickHandler} name="AC" color="e0e0e0" />
+      <Button onClick={clickHandler} name="+/-" color="e0e0e0" />
```

Comment:
- [ ] **[OPTIONAL]** We can dynamically create these buttons with `map` function by mapping through 2Dimensional array of buttons, like this:
```js
const buttons = [
    ['AC', '+/-', '%', '÷'], 
    ['7', '8', '9', 'X'], 
    ['4', '5', '6', '-'], 
    ['1', '2', '3', '+'], 
    ['0', '.', '=']
];
```

Then you can use map to loop through the outer array, and another map to loop through the inner arrays.